apps/testrunner/test_suites/selenium_tests/selenium_timer.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import logging
+
+logger = logging.getLogger(__name__)
+
+class SeleniumTimer(object):
+
+    def __init__(self, driver):
+        self.driver = driver
+        self.measurements = []
+
+    def start(self):
+        self.start_timestamp = self.driver.execute_script("return new Date().getTime()")
+
+    def mark_measurement_point(self, url):
+        navigation_start = self.driver.execute_script("return window.performance.timing.navigationStart")
```

Comment:
Can we get the entire window.performance.timing object at once and select properties at python side? Communication with web browser has for sure an overhead.